tests/api/ApiTests.cpp

```diff
@@ -12,10 +12,10 @@ static void define_variable_attributes(int file_id, std::string const& variable_
     auto error_code = ugridapi::ug_name_get_long_length(name_long_length);
     ASSERT_EQ(ugridapi::UGridioApiErrors::Success, error_code);
 
-    std::unique_ptr<char> const variable_name_ptr(new char[name_long_length]);
+    std::unique_ptr<char[]> const variable_name_ptr(new char[name_long_length]);
```

Comment:
i propose we write a task to remove operator new in the unit tests as much as possible (using strings, vectors)

Reply:
done